db/schema.rb

```diff
@@ -10,10 +10,27 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_07_21_213350) do
+ActiveRecord::Schema[7.0].define(version: 2023_08_25_191958) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "saved_recipies", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.integer "api_recipie_id", null: false
+    t.boolean "favorited", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_saved_recipies_on_user_id"
+  end
+
+  create_table "scheduled_recipies", force: :cascade do |t|
+    t.bigint "saved_recipie_id", null: false
+    t.datetime "scheduled_date"
```

Comment:
We probably dont want this to be nullable ... Is there a situation you thought of that I'm not considering where this might need to be nullable vs just deleted? Or are you thinking of it as a way to "delete" without actually deleting?